introduction/modules/dashboard-introduction.md

```diff
@@ -0,0 +1,18 @@
+# What are dashboards
+
+Dashboards can provide the user with statistical information.
+Using this information, the user will be enabled to prioritize and focus on specific tasks.
+
+A dashboard contains a set of widgets that display data in a customized way. 
+These dashboards and widgets can be configured by an administrator accordingly.
+
+## Concepts
+
+* **Dashboard** A container for any number of widgets. 
+    Multiple dashboards can be configured, which will be shown to the user as tabs.
+* **Widget** Presents data from a datasource using a display-type to the user.
+* **Data-source** Retrieves data from any desired source. 
+    Every datasource can publish output features like `value`, `total` or `values`.
```

Comment:
```suggestion
* **Data-source** Every dashboard widget retrieves its data from a data-source. 
```

I don't think people need to know about response body fields: 'value', 'total', 'values'

Reply:
You're probably right that the introduction is not the right place for that information.